airflow/utils/task_group.py

```diff
@@ -398,6 +389,8 @@ def child_id(self, label):
     @property
     def upstream_join_id(self) -> str:
         """
+        # TODO: D205-ify this one
```

Comment:
Advice?

Reply:
how about "Creates a unique id to represent upstream dependencies of this TaskGroup"?